src/components/Audit.vue

```diff
@@ -420,7 +420,12 @@ export default {
       return `--percent: ${pfulfilled}%; --bar-color: ${pcolor}; --bg-color: #fff`;
     },
     deleteReq: function (req) {
-      const reqName = req['list-id'].substring(0, req['list-id'].indexOf('.reql'));
+      let reqName;
+      if (req['list-id'].includes('.reql')) {
+        reqName = req['list-id'].substring(0, req['list-id'].indexOf('.reql'));
```

Comment:
Are there any situations where it takes this branch?  I tested a bunch and it seemed like it never had the .reql

Reply:
I think you're right that it never has .reql